src/main/scala/org/broadinstitute/clio/dataaccess/CachedServerStatusDAO.scala

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.clio.dataaccess
+
+import org.broadinstitute.clio.model.ServerStatusInfo
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * Stores the server status in a synchronized mutable variable.
+  */
+class CachedServerStatusDAO private() extends ServerStatusDAO {
```

Comment:
@rtitle is currently adding (or done adding?) server status to rawls. It might be worth looking at what he's done, to avoid work duplication, even if we don't have a way to share code across projects.